openapi.json

```diff
@@ -45,13 +45,35 @@
               "transactions.read": "Access to list and view accounts and transactions",
               "transactions.write": "Access to create, update and delete transactions",
               "categories.read": "Access to view categories",
-              "categories.write": "Ability to edit and delete categories"
+              "categories.write": "Ability to edit and delete categories",
+              "attachments.read": "Access to view attachments",
+              "attachments.write": "Ability to create, update and delete attachments"
             }
           }
         }
       }
     },
     "responses": {
+      "201": {
```

Comment:
@Concetta there's a few issues here:

* Both responses are using the Error schema, but they're not errors
* Since any data can be returned, we can't point to a particular schema for these. For the generic errors we have specified, the response is always an Error schema, so we can say that definitively.
* We don't need these; these globally defined responses are for common errors, not for all responses. Look at existing endpoints for 201 and 204 for how we respond with those, and remove these.